client/src/components/forms/field-error.component.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { describe, expect, it } from 'vitest';
+import { FieldError } from './field-error.component';
+import { render, screen } from '~/test/utils';
+
+describe('form-error', () => {
+  it('should render field-error on the page', () => {
+    render(<FieldError>Test Error</FieldError>);
+    expect(screen.getByText('Test Error')).toBeInTheDocument();
+  });
```

Comment:
You are testing that a bit of text appears and that is a great first test. Maybe consider another test that tests more complicated HTML as well. So instead of passing just text, we wrap the text in a div with 2 spans, on for `Test` and the other for `Error`. This is a fake layout of course, but what it would test is that the text still appears, even though it is in different HTML elements.